src/one_hot_encoding.jl

```diff
@@ -32,7 +32,7 @@ function _apply(x, encoding::OneHotEncoding; kwargs...)
 
     results = zeros(Int, length(x), n_categories)
 
-    for (i, value) in enumerate(x)
+    @views for (i, value) in enumerate(x)
```

Comment:
Why @views at this level? Is it because it should be done whenever iterating the data, and this is the only internal _apply that does so?

Reply:
Not necessarily every time but since we are indexing into `results` and assigning new values I figured using `@views` might help here. I could be wrong though.